src/Components/Search/__tests__/SearchBar.test.tsx

```diff
@@ -0,0 +1,44 @@
+import Input from "Components/Input"
+import { SearchBar } from "Components/Search/SearchBar"
+import { MockBoot } from "DevTools/MockBoot"
+import { mount } from "enzyme"
+import React from "react"
+
+const searchResults = {
+  search: {
+    edges: [
+      {
+        node: {
+          displayLabel: "Percy Z",
+          href: "/cat/percy-z",
+          searchableType: "Cat",
+          id: "percy-z",
+        },
+      },
+    ],
+  },
+}
+
+describe("SearchBar", () => {
+  const getWrapper = viewer => {
+    return mount(
+      <MockBoot breakpoint="xl">
+        <SearchBar
+          viewer={viewer as any}
+          relay={{ environment: {}, refetch: () => null } as any}
```

Comment:
We're using `refetch` (and that's not something we're testing currently, so this is just a stub for now).